src/js/media-seek-backward-button.js

```diff
@@ -1,17 +1,15 @@
 import MediaChromeButton from './media-chrome-button.js';
 import { window, document } from './utils/server-safe-globals.js';
 import { MediaUIEvents, MediaUIAttributes } from './constants.js';
-import { verbs } from './labels/labels.js';
-import { getSlotted, updateIconText } from './utils/element-utils.js';
+import { updateAriaLabel, updateSeekIconValue } from './utils/seek.js';
 
 export const Attributes = {
-  SEEK_OFFSET: 'seekoffset'
+  SEEK_OFFSET: 'seekoffset',
 };
 
 const DEFAULT_SEEK_OFFSET = '30';
```

Comment:
question(non-blocking): With all the attention to types (TS and JS), should this be a number?

Reply:
I changed it to a string at some point because TSC complained because it expected `setAttribute` to be given a string and all the usages of this variable are in setAttribute.

Reply:
For "correctness" sake I'd prefer it to be a number but I'm going to leave it as-is for simplicity and less casting everywhere.